projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/asm/b2c/customer-list.e2e-flaky.cy.ts

```diff
@@ -4,16 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as asm from '../../../../helpers/asm';
-import * as checkout from '../../../../helpers/checkout-flow';
-import { clearAllStorage } from '../../../../support/utils/clear-all-storage';
+import * as asm from '../../../helpers/asm';
+import * as checkout from '../../../helpers/checkout-flow';
+import { getSampleUser } from '../../../sample-data/checkout-flow';
+import { clearAllStorage } from '../../../support/utils/clear-all-storage';
 
 context('Assisted Service Module', () => {
   before(() => {
     clearAllStorage();
   });
 
   describe('ASM Customer list', () => {
+    const productCode = '479742';
```

Comment:
please also add e2e test for b2b

Reply:
Add ticket to track it.
https://jira.tools.sap/browse/CXSPA-3385